addons/mass_mailing_crm/views/mailing_mailing_views.xml

```diff
@@ -22,4 +21,17 @@
     		</xpath>
         </field>
     </record>
-</odoo>
\ No newline at end of file
+
+    <record id="crm_lead_view_search" model="ir.ui.view">
```

Comment:
Redefining the whole view for this seems a bit overkill.

Can't we simply turn on:
"search_default_month": 1
In the context?
(Yes, for some reason that filter name is "month" ...)

I know it will not group by day as requested but at least it's consistent with the search view.
Same applies for the other inherited views.

Maybe something to re-discuss with @LNAodoo ?

cc @tde-banana-odoo @dbeguin

Reply:
Hello again @SwagSamaSempai 

After discussing with Thibault and Luc, we prefer this approach:
https://github.com/odoo/odoo/pull/79166/files#diff-35ba5c308cdb8d8b3dda81053a97d9d12fa686b7fce6241a0db7629b87740b9dR724

Using a bit of context keys.
It does the same thing but reduces the code and avoid extra views.
(Context keys naming should look like: "registration_view_hide_group_by_create_date_week", adapted with view name and granularity of course).

Thanks!